sunpy/map/tests/test_mapbase.py

```diff
@@ -1181,6 +1181,27 @@ def test_quicklook(aia171_test_map):
         assert aia171_test_map._repr_html_() in html_string
 
 
+# requires dask array to run properly
+def test_dask_array(aia171_test_map):
+    dask_array = pytest.importorskip('dask.array')
+
+    da = dask_array.from_array(aia171_test_map.data, chunks=(1, 1))
+    pair_map = sunpy.map.Map(da, aia171_test_map.meta)
+
+    # Check that _repr_html_ functions for a dask array
+    html_string = pair_map._repr_html_()
+    assert isinstance(html_string, str)
```

Comment:
Wouldn't this test pass on our main branch right now? The issue isn't necessarily whether it functions, but whether it pulls the data array into memory.

Reply:
One option that might be a little hacky: you could call `_repr_html` with `compute_dask=True` and `False` and then assert that they are unequal for the case of the map that holds a Dask array.

Reply:
Changed test as suggested